app/controllers/notifications_controller.rb

```diff
@@ -41,8 +41,8 @@ def index
 
       pager.replace(result)
     end
-    @notifications.each do |n|
-      n.note_html = render_to_string( :partial => 'notify_popup_item', :locals => { :n => n } )
+    @notifications.each do |note|
+      note.note_html = render_to_string( :partial => 'notification.html', :locals => { :note => note } )
```

Comment:
Maybe. But this PR is definitely too big. I think the port to Handlebars should be made in another PR.